core-api/src/main/java/com/optimizely/ab/notification/NotificationCenter.java

```diff
@@ -176,12 +186,12 @@ public int addNotificationListener(NotificationType notificationType, Notificati
         }
 
         for (NotificationHolder holder : notificationsListeners.get(notificationType)) {
-            if (holder.notificationListener == notificationListener ) {
+            if (holder.notificationListener == notificationListener) {
                 logger.warn("Notification listener was already added");
                 return -1;
```

Comment:
I see that returning `-1` is existing behavior. It's misleading and worth changing IMO. Maybe a later PR, but maybe add a TODO